main.go

```diff
@@ -178,21 +180,37 @@ func upload(config *Config, fileStat *FileStat, dryrun bool, logger *Logger) err
 		}
 	}()
 
+	// this is the content-type that http.DetectContentType() fall backs to
```

Comment:
Redundant comment. It's pretty clear that's what it does.

Reply:
roger, done.